tensorflow_addons/image/median_filter_2D.py

```diff
@@ -0,0 +1,86 @@
+import tensorflow as tf
+from tensorflow.python.ops.image_ops_impl import _Assert3DImage
+
+
+
+def median_filter_2D(input, filter_shape=(3, 3)):
```

Comment:
Please decorate this with `@tf.function` so we can leverage all optimizations possible in graph mode.

Reply:
solved